.github/workflows/tests-with-minimum-versions.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+  pull_request: {}
```

Comment:
TODO: Remove this temporary change before merging.

```suggestion
```

Reply:
I confirmed that this PR resolves the error.
https://github.com/optuna/optuna/actions/runs/4201654625/jobs/7288874421